index.js

```diff
@@ -148,7 +156,7 @@ function math_block(state, start, end, silent){
     + state.getLines(start + 1, next, state.tShift[start], true)
     + (lastLine && lastLine.trim() ? lastLine : '');
     token.map = [ start, state.line ];
-    token.markup = '$$';
+    token.markup = startDelim;
```

Comment:
what happens to the token markup when the start and end character isn't the same. Is it even supported?

See [the api documentation](https://markdown-it.github.io/markdown-it/#Token.prototype.markup)